File: src/main/java/tom/com/Guess/Game.java
package tom.com.Guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secret = random.nextInt(10) + 1;
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (number != secret) {
            System.out.print("The number you guess is? ");
            number = scanner.nextInt();
            if(number>secret){
                System.out.println("lower");
            }else if(number<secret){
                System.out.println("higher");
            }else {
                System.out.println("you got it, the number is "+ secret);
            }
        }
    }
}

File: src/main/kotlin/com/tom/Game.kt
package com.tom

import java.util.*

fun main(args: Array<String>) {
    val secret = Random().nextInt(10)+1
    println(secret)
  //  val scanner = Scanner(System.`in`)
    var number = 0
    while (number != secret){
    println("please enter the number")
//    number = scanner.nextInt()
        number = readLine()!!.toInt()
        if (number > secret){
            println("lower")
        }else if (number <secret){
            println("higher")
        }else{
            println(" you got is, the number is $secret")
        }
}
}